cocos/platform/CCFileUtils.h

```diff
@@ -809,6 +809,14 @@ class CC_DLL FileUtils
      *  @return File paths in a string vector
      */
     virtual std::vector<std::string> listFiles(const std::string& dirPath) const;
+
+    /**
+     * List all files in a directory async, off of the main cocos thread.
+     *
+     * @param dirPath The path of the directory, it could be a relative or an absolute path.
+     * @param callback The callback to be called once the list operation is complete. Will be called on the main cocos thread.
+     */
```

Comment:
It seems the error happens in binding codes. Can add comments to disable binding like:

@lua NA
@js NA